datawig/simple_imputer.py

```diff
@@ -87,8 +87,8 @@ class SimpleImputer:
     """
 
     def __init__(self,
-                 input_columns: List[str],
-                 output_column: str,
+                 input_columns: List[str] = None,
+                 output_column: str = "",
```

Comment:
We should require these two arguments, this could be confusing to users why you can construct a `SimpleImputer` without any arguments.

Maybe we can make saving and loading easier with a static method instead?

Reply:
Even if we make it fail gracefully, saying "you need to specify input/output columns."?

I'm not sure how to make it a static method of the SimpleImputer. That would always require initializing a SimpleImputer instance first, wouldn't it?

Slightly clunky alternative, have a simple utils function:

```
def load_simple_imputer(output_path: str) -> Any:

    from datawig.simple_imputer import SimpleImputer
    imputer = SimpleImputer([], "").load(output_path)

    return imputer
```